plugins/CoreAdminHome/javascripts/generalSettings.js

```diff
@@ -123,6 +123,13 @@ $(document).ready(function () {
                 var frameContent = $(uploadFrame.contents()).find('body').html();
                 frameContent = $.trim(frameContent);
 
+                if ('0' === frameContent) {
+                    $('.uploaderror').show();
+                    setTimeout(function(){
+                        $('.uploaderror').fadeOut();
```

Comment:
Can we maybe keep the error message visible and instead when uploading another image remove a possible displayed notification? Might be hard to read within 5 seconds but not sure how we handle this in other places